en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,117 @@ often working alongside Core Committers.
 
 CMS Squad members have write access to core repositories in order to work effectively with GitHub issues. They are expected to use those permissions with good judgement for merging pull requests.
 
+### Peer Reviewers
+
+[info]
+This is a new role, and is currently in a trial period. It is likely to change over time as we learn what works and what doesn't.
+[/info]
+
+This role is made up of community members who have demonstrated a willingness to improve Silverstripe CMS through their contributions to the open source supported modules. It empowers community members to have a more active role in ensuring pull requests get merged, and builds more momentum for community-created pull requests.
+
+While anyone in the community can review a pull request, only reviews from maintainers have weight of authority - and only maintainers can meaningfully approve changes. The Peer Reviewer role ensures members of the community have that authority.
+
+#### Responsibilities {#peer-reviewer-responsibilities}
+
+* Review pull requests, ensuring documented processes and best practices are being followed
+  * Refer to [how to review pull requests](/contributing/triage_resources/#how-to-review-pull-requests) and [contributing code](/contributing/code/)
+* Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
+  * If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
+* Escalate pull requests to Core Committers if the PR is very complex and you don't feel confident reviewing it
+  * If another reviewer feels confident, they can offer to review it instead
```

Comment:
Swap around the points. Talking about folks being rude as the first escalation point feels like going into this already with a pre-negative mindset.

```suggestion
* Escalate pull requests to Core Committers if the PR is very complex and you don't feel confident reviewing it
  * If another reviewer feels confident, they can offer to review it instead
* Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
  * If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
```

Reply:
Done